cli/dcoscli/analytics.py

```diff
@@ -107,7 +107,8 @@ def _segment_request(path, data):
         requests.post('{}/{}'.format(SEGMENT_URL, path),
                       json=data,
                       auth=HTTPBasicAuth(key, ''),
-                      timeout=1)
+                      timeout=1,
+                      proxies=util.get_proxy_dict())
```

Comment:
To avoid reading the config file 3 times instead of just once. Let's either pass the `conf` object into `_segment_request` or pass the proxy information into it.